pkg/kafka/request.go

```diff
@@ -217,12 +223,15 @@ func ReadRequest(reader io.Reader) (*RequestMessage, error) {
 	case proto.OffsetFetchReqKind:
 		req.request, err = proto.ReadOffsetFetchReq(buf)
 	default:
-		log.WithField(fieldRequest, req.String()).Debugf("Unknown Kafka request API key: %d", req.kind)
+		if flowdebug.Enabled() {
+			log.Debugf("Unknown Kafka request API key: %d in %s", req.kind, req.String())
```

Comment:
Same as comment above, please consider using structured log, (but `if flowdebug.Enabled` line is probably a good call).

Reply:
See above :-)